src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -145,7 +145,7 @@ describe('ReactDOMComponent', () => {
     it('should warn for unknown prop', () => {
       spyOn(console, 'error');
       var container = document.createElement('div');
-      ReactDOM.render(<div foo="bar" />, container);
+      ReactDOM.render(<div foo={() => {}} />, container);
       expectDev(console.error.calls.count(0)).toBe(1);
       expectDev(normalizeCodeLocInfo(console.error.calls.argsFor(0)[0])).toBe(
         'Warning: Unknown prop `foo` on <div> tag. Remove this prop from the element. ' +
```

Comment:
Sounds like a great idea to me. I've also taken a quick pass at the intro of the related documentation (07af3db).